src/software/thunderscope/gl/layers/gl_tactic_layer.py

```diff
@@ -8,7 +7,6 @@
 
 from proto.import_all_protos import *
 from software.py_constants import *
-import software.thunderscope.constants as constants
 from software.thunderscope.constants import Colors
 
 from software.thunderscope.gl.graphics.gl_circle import GLCircle
```

Comment:
Nit: There seems to be a lot of unused imports in this file. 
All of `software.thunderscope.gl.graphics.*` can be removed + `ThreadedUnixListener`

Reply:
removed imports